se.trixon.Mapollage.yml

```diff
@@ -0,0 +1,53 @@
+id: se.trixon.Mapollage
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: start-mapollage
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
```

Comment:
this is a very broad filesystem permission

Reply:
Yes, as I wrote in the form its purpose is so that the user can add photos from any location.